### DISCUSSION TITLE
pycom wipy pysense pytrack firmware

### DISCUSSION THREAD

**Discussion by priisdk** [micropython/micropython#9788] [General]
I have a pycom Wipy (WIPY3.0 IX 22263 WIPY3R) board and two expansion boards, pytrack v1.1 and pysense v1.0.
I have updated the firmware on the expansion boards, carefully following the instructions here:
https://www.youtube.com/watch?v=FkycTZvj-ss
I have also updated the firmware on the Wipy board following the instructions here:
https://www.youtube.com/watch?v=TEfiMnkvyyM
(device type: WiPy, Type: pybytes, version 1.20.2.r6, file system: LittleFS, empty boxes at RESET, COFIG partition and NVS partition).

Before I did the updating everything seemed to work - I could read barometric pressure, temperature, acceleration etc. However, after the updating I cannot even connect to any editor. For example, the Mu editor gives the following reply when connecting the board to my PC:
"Could not find an attached devise. Please make sure the device is plugged into this computer. It must have a version of MicroPython (or CircuitPython) flashed onto it before the REPL will work. Finally, press the device's reset button and wait a few seconds before trying again."
When I connect the board's led lights red briefly, followed by blinking blue (heartbeat led).
What can I do to make my board work again?

Poul Riis

**Comment by robert-hh** [comment]
This is the wrong place for your question. You should ask it at the Pycom forum: https://pycom.forum.io
But, as initial hint, please check if there is a USB serial device visible on your computer when you plug in the expansion board. If not, you have to repeat the firmware update for the expansion boards. For firmware updates, there are as well good instructions at the Pycom documentation site https://docs.pycom.io/updatefirmware/

**Reply by scruss** [reply]
[pycom.forum.io](http://pycom.forum.io/) — like the rest of the Pycom company — is down.

**Reply by robert-hh** [reply]
https://forum.pycom.io/ is still there. As well as https://pycom.io/

**Reply by robert-hh** [reply]
But the certificate for the forum site expires on Nov 23, and that for the Company web site on Dec 14. If no one renews them (which is likely to happen), then a default configured browser will refuse to connect.

**Reply by sosi-deadeye** [reply]
Update: The content of the page cannot be displayed

**Reply by robert-hh** [reply]
But that‘s only for forum.pycom.io. If i write the full https://forum.pycom.io it works for me.
